### PR TITLE
Add more detail to user_preferences explainer

### PR DESCRIPTION
**Pull request by loubrett**

### CODE REVIEW

File: user-preferences-explainer.md [hunk 1]
```diff
@@ -32,6 +34,10 @@ For the `user_preferences` member, the acceptable keys for the `ManifestOverride
         *   `src`
         *   `type`
 
+For user_preferences which the host OS supports, implementers should make the relevant overrides available to the OS. For example, if an OS supports dark mode and an app has specified icon overrides for dark mode, the implementer should download these icons in addition to the icons they already download.
```
Comment (tomayac) [inline user-preferences-explainer.md]:
```suggestion
For `user_preferences` which the host OS supports, implementers should make the relevant overrides available to the OS. For example, if an OS supports dark mode and an app has specified icon overrides for dark mode, the implementer should download these icons in addition to the icons they already download.
```

File: user-preferences-explainer.md [hunk 2]
```diff
@@ -32,6 +34,10 @@ For the `user_preferences` member, the acceptable keys for the `ManifestOverride
...
+ 
+SVG icons natively support user_preferences through CSS. Therefore SVG manifest icons (if supported) should also be rendered under all the OS supported user_preferences.
```
Comment (tomayac) [inline user-preferences-explainer.md]:
```suggestion
SVG icons natively support `user_preferences` through CSS. Therefore SVG manifest icons (if supported) should also be rendered under all the OS supported `user_preferences`.
```

**Review by tomayac — COMMENTED** [review]
For consistency `user_preferences` should be written in a code font. In the remarked cases, though, you probably rather want to speak of "user preferences" in prose text.

**Comment by aarongustafson** [comment]
I'm currently out sick, but will take a look by next Monday, when I hope to be in better shape to review.

**Review by aarongustafson — APPROVED** [review]
LGTM. Thanks Lou!

**Pull request merged**